test/abi.inputParser.js

```diff
@@ -294,13 +244,13 @@ describe('abi', function() {
 
         });
 
-        it('should parse input string', function () {
+        it('should parse input fixed bytes type', function () {
 
             // given
             var d = clone(description);
 
             d[0].inputs = [
-                { type: "string" }
```

Comment:
But in solidity there is no strings type, so bytes would probagbly be more correct